### DISCUSSION TITLE
Add contribute history on profile for every user

### DISCUSSION THREAD

**Discussion by Haythamasalama** [community/community#69527] [Mobile]
Add contributes history for every user on profile page and our profile.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.132.0 (110686114)
Device: iPhone 13 (iOS 17.0)
</details>

**Comment by elisealix22** [comment]
Hi @Haythamasalama , thank again for another great feature request! I created an issue to track this one internally. Please keep 'em coming as you think of more.
